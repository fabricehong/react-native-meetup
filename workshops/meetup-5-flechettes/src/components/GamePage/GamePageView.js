import React, {PureComponent} from 'react';
import {View} from 'react-native';

export class GamePageView extends PureComponent {
  render() {
    return (
      <View/>
    );
  }
}

export default GamePageView;

