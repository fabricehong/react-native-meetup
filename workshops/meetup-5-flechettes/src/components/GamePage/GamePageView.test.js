import React from 'react';
import GamePageView from './GamePageView';
import renderer from 'react-test-renderer';

describe('GamePageView', () => {
  test('should renders correctly', () => {
    const tree = renderer.create(<GamePageView/>).toJSON();
    expect(tree).toMatchSnapshot();
  });
});