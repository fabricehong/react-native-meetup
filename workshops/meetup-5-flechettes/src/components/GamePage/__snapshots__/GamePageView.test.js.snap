// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`GamePageView should renders correctly 1`] = `<View />`;
